crafters/audio/AudioMonophoner/README.md

```diff
@@ -1,3 +1,43 @@
 # AudioMonophoner
 
-makes the audio signal monophonic on doc-level. 
\ No newline at end of file
+**AudioMonophoner** makes the audio signal monophonic on doc-level.
+
+The supported sound formats are:
+WAV, MP3, OGG, AU,FLAC, RAW, AIFF, AIFF-C, PAF, SVX, NIST, VOC, IRCAM, W64, MAT4, MAT5, PVF, XI, HTK, SDS, AVR, WAVEX, SD2, CAF, WVE, MPC2K, RF64.
+
+## Usage
+
+The **AudioMonophoner** executor uses the `librosa` library to convert the input `blob` (`ndarray` representation of the audio signal) to a dictionary of `monophonic` audio signal. 
+
+We use Pod images in several ways:
+
+1. Run with Docker: `docker run`
+   ```bash
+    docker run jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
+    ```
```

Comment:
The pod won't be usable.
```suggestion
1. Run with Docker: `docker run`
   ```bash
    docker run jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
    ```
```

Reply:
Binding the container port here